gossip/src/low_pass_filter.rs

```diff
@@ -0,0 +1,62 @@
+//! Fixed-point IIR filter for smoothing `alpha` updates.
+//!
+//! Implements:
+//!   alpha_new = K * target + (1 - K) * previous
+//!
+//! All math is unsigned integer fixed-point with `SCALE = 1000`
+//!
+//! The filter constant K is derived from:
+//!     K = W_C / (1 + W_C), where Wc = 2π * Fs / Tc
+//!     Fc = 1 / TC  (cutoff frequency)
+//!     Fs = 1 / refresh interval
+use std::num::NonZeroU64;
+
+// Fixed point scale for K and `alpha` calculation
+pub const SCALE: NonZeroU64 = NonZeroU64::new(1000).unwrap();
```

Comment:
What is the benefit of having this as NonZero type if this is a constant anyway?

Reply:
It may further be preferable to have this as a power of two for performance reasons.